### PR TITLE
skip saving unchanged asset while training

### PR DESCRIPTION
**Pull request by starain-pactera**
add logical to skip saving unchanged asset while training

### CODE REVIEW

**Review by yongbing-chen — DISMISSED** [review]

File: src/react/components/pages/train/trainPage.tsx [hunk 1]
```diff
@@ -419,42 +421,45 @@ export default class TrainPage extends React.Component<ITrainPageProps, ITrainPa
...
-                    }
-                });
-                if (!isUpdated&&assetMetadata.asset.labelingState === AssetLabelingState.ManuallyLabeled
-                    && assetMetadata.labelData?.labels?.findIndex(label => label.confidence
-                        || label.originValue
-                        || label.revised
-                        || label.value?.findIndex(item => item["confidence"]) < 0) < 0
-                ) {
-                    return;
+        const assetValues=Object.values(allAssets).filter(asset => asset.labelingState !== AssetLabelingState.Trained)
```
Comment (stew-ro) [inline src/react/components/pages/train/trainPage.tsx:424]: add spacing

Comment (stew-ro) [reply src/react/components/pages/train/trainPage.tsx:424]: "assetValues = Object..."

File: src/react/components/pages/train/trainPage.tsx [hunk 2]
```diff
@@ -329,6 +328,9 @@ export default class TrainPage extends React.Component<ITrainPageProps, ITrainPa
                     const newMeta = await assetService.save({ ...metadata });
                     newAssets[asset.id] = newMeta.asset;
                 }
+                else {
+                    newAssets[asset.id]=newAsset;
```
Comment (stew-ro) [inline src/react/components/pages/train/trainPage.tsx:332]: add spacing

Comment (starain-pactera) [reply src/react/components/pages/train/trainPage.tsx]: Done, thanks.

**Review by stew-ro — APPROVED** [review]

**Pull request merged**